frontend_vue/src/components/base/BaseLabelArrow.vue

```diff
@@ -0,0 +1,59 @@
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <BaseLabel
+    :id="id"
+    :required="required"
+    class="relative mb-8"
+  >
+    <span v-html="labelArrowed"></span>
+  </BaseLabel>
+</template>
+
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+
+const props = defineProps<{
+  id: string;
+  label: string;
+  arrowVariant?: 'one' | 'two';
+  arrowWordPosition?: number;
```

Comment:
heh, this component is very cute

to over-engineer this even further you could even see this being an array (not really feedback)

Reply:
don't tempt me...!